src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -1,29 +1,71 @@
 package nl.novi.techiteasy.controllers;
-
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-
+import nl.novi.techiteasy.exceptions.RecordNotFoundException;
+import nl.novi.techiteasy.models.CreateTvRequest;
+import nl.novi.techiteasy.models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
 
 @RestController
+@RequestMapping("/televisions")
 public class TelevisionController {
+    private final ArrayList<Television> televisionDataBase = new ArrayList<>();
+    private final AtomicInteger idCounter = new AtomicInteger(1);
+    @PostMapping
+    public ResponseEntity<Television> create(@RequestBody CreateTvRequest model) {
+        int id = idCounter.getAndIncrement();
+        Television result = new Television(id, model.name, model.price);
+        this.televisionDataBase.add(result);
+        return ResponseEntity.created(null).body(result);
+    }
 
-    ArrayList<String> television = new ArrayList<>();
-    @PostMapping("/add")
-    public void SaveTelevision(@RequestParam String tv) {
-        this.television.add(tv);
+    @GetMapping("/get")
```

Comment:
Een GetMapping zegt eigenlijk al dat het voor een get request is. het is dan niet heel conventioneel om hier nog een /get te gebruiken om een specifieke tv op te halen.